src/main/java/org/opensearch/knn/plugin/script/KNNScoringUtil.java

```diff
@@ -294,11 +295,7 @@ public static float lInfNorm(List<Number> queryVector, KNNVectorScriptDocValues
      */
     public static float innerProduct(float[] queryVector, float[] inputVector) {
         requireEqualDimension(queryVector, inputVector);
```

Comment:
I think this check is present in Lucene code, we can remove it just like we removed from l2Squared

Reply:
Sure, I will remove it, also how do I add this to changelog.md. I have read the CONTRIBUTING.md but am still quite confused about how to do it?

Reply:
@navneet1v Yes, I would love to start with something small.